packages/components/src/components/as-histogram-widget/as-histogram-widget.tsx

```diff
@@ -563,13 +563,15 @@ export class HistogramWidget {
 
     this.barsContainer = drawService.renderPlot(this.container);
 
+    const tooltipFormatter = this.tooltipFormatter || this._tooltipFormatter;
```

Comment:
I think this is never going to be `this._tooltipFormatter`, this.tooltipFormatter has a default value of this.formatter.

Reply:
If you think this behaviour is better (showing the start + the value) we should change it in this.formatter, but keep in mind that one does some differentiation between categorical data and numerical data.

BTW, while this is a change in the default behaviour of the component I think it doesn't qualify as a breaking change, since users are able to change it in case it's not exactly what they want.